roles/webhooks/main.yml

```diff
@@ -0,0 +1,11 @@
+# Copy local webhooks into webhooks folder
+- name: copy webhook plugins
+  copy:
+    src: "{{ webhook_plugins_src_dir }}/{{ item }}"
+    dest: "{{ webhook_plugins_dest_dir }}/{{ item }}"
+    owner: root
+    group: root
+    mode: 0755
+  loop: "{{ webhook_plugins }}"
```

Comment:
It would make more sense for webhook_plugins to be `[]` instead of `null`, could leave out the `when` in this block.